components/config/tests/prefs.rs

```diff
@@ -1,333 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This testing code has mainly moved to servoshell as that is what handles setting preferences via JSON now.